src/dswx_sar/metadata.py

```diff
@@ -266,24 +337,54 @@ def _get_general_dswx_metadata_dict(cfg, product_version=None):
 
     # save datetime 'YYYY-MM-DDTHH:MM:SSZ'
     dswx_metadata_dict['PROCESSING_DATETIME'] = datetime.now().strftime("%Y-%m-%dT%H:%M:%SZ")
-    dswx_metadata_dict['POLARIZATION'] = cfg.groups.processing.polarizations
 
     return dswx_metadata_dict
 
 
-def gather_rtc_files(rtc_dirs):
+def gather_rtc_files(rtc_dirs, pol):
     """
     Given directories containing RTC files, gather all h5 files.
     """
-    h5_files = []
+    tif_files = []
     for rtc_input_dir in rtc_dirs:
-        rtc_h5_file = glob.glob(os.path.join(rtc_input_dir, '*.h5'))
-        if rtc_h5_file:
-            h5_files.append(rtc_h5_file[0])
-    return h5_files
+        rtc_tif_file = glob.glob(os.path.join(rtc_input_dir, f'*{pol}*.tif'))
```

Comment:
```suggestion
        rtc_tif_file = glob.glob(os.path.join(rtc_input_dir, f'*{pol.upper()}*.tif'))
```

Another suggestion: what do you think about using [glob.iglob](https://docs.python.org/3/library/glob.html#glob.iglob)?